src/pages/home/Home.page.tsx

```diff
@@ -132,7 +139,7 @@ export default function Home() {
 
   return (
     <>
-      <Text>Home</Text>
+      <Text>Home!</Text>
       <Button onClick={toggleColorMode}>Toggle theme</Button>
       <StatsWithIcons />
```

Comment:
Change - The `toggleColorMode` method is called directly within the `onClick` event. Ensure that this function is properly defined and imported in your component to prevent runtime errors. Verify its integration to deliver the desired functionality.